lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -50,6 +50,7 @@ use constant FINAL_STATES     => (DONE,    CANCELLED);
 use constant {
     NONE               => 'none',
     PASSED             => 'passed',
+    SOFTFAILED         => 'softfailed',
```

Comment:
See that comment a few lines earlier where it says "The state and results constants are duplicated in the Python client: if you change them or add any, please also update const.py." ? Well, I don't see a pull request for openQA-python-client :/ good thing I spotted this...